.github/workflows/cd.yml

```diff
@@ -7,10 +7,8 @@ on:
         required: true
         type: choice
         options:
-          - frontend-analytics
-          - frontend-language-toggle
-          - frontend-passthrough-headers
           - "@govuk-one-login/alpha-component"
+          - frontend-vitals-monitoring
```

Comment:
Was this amended in the end? `@govuk-one-login/frontend-vitals-monitoring`

Reply:
Ah yeah sorry only did it in my other repo before. Updated here now too 👍